### PR TITLE
Update Chef Infra Client to 15.10.12 & Fauxhai to 8.0

### PR DESCRIPTION
**Pull request by tas50**
It's important that we get a somewhat recent release into the current
channel to resolve bootstrapping problems that customers are having.

Fauxhai is updated so we can test on new platforms with ChefSpec

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: www/layouts/shortcodes/ws_chef_shell_debug_existing_recipe.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ loading configuration: none (standalone session)
 Session type: standalone
 Loading.............done.
 
-Welcome to the chef-shell 15.8.23
+Welcome to the chef-shell 15.10.12
```
Comment (kagarmoe) [inline www/layouts/shortcodes/ws_chef_shell_debug_existing_recipe.md:11]: Obviously not a blocker, but you don't need an indefinite article:

```suggestion
Welcome to chef-shell 15.10.12
```

File: www/layouts/shortcodes/ws_chef_shell_example_hello_world.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ bin/chef-shell
   Session type: standalone
   Loading.............done.
 
-  Welcome to the chef-shell 15.8.23
+  Welcome to the chef-shell 15.10.12
```
Comment (kagarmoe) [inline www/layouts/shortcodes/ws_chef_shell_example_hello_world.md:19]:
```suggestion
  Welcome to chef-shell 15.10.12
```

File: www/layouts/shortcodes/ws_chef_shell_debug_existing_recipe.md (hunk 1 above)
Comment (tas50) [reply www/layouts/shortcodes/ws_chef_shell_debug_existing_recipe.md:11]: That text is dated now, but it's the actual text from the command

Comment (kagarmoe) [reply www/layouts/shortcodes/ws_chef_shell_debug_existing_recipe.md:11]: I assumed--it just stands out.

**Pull request merged**